package/SUSEConnect.spec

```diff
@@ -26,7 +26,6 @@ Requires:       zypper >= 1.11.32
 Conflicts:      suseRegister, yast2-registration < 3.1.129.7
 
 Obsoletes:      ruby2.1-rubygem-suse-connect < %{version}
-Provides:       %{rb_default_ruby_suffix}-rubygem-suse-connect = %{version}
```

Comment:
We discussed it with @digitaltom and then I tested this rpm against old two-package configuration, and it turns out to be redundant